spec/bourbon/css3/font_face_spec.rb

```diff
@@ -24,7 +25,8 @@
                 "format(\"woff2\"), url(\"/assets/fonts.woff\") " +
                 "format(\"woff\"), url(\"/assets/fonts.ttf\") " +
                 "format(\"truetype\"), url(\"/assets/fonts.svg#Verdana\") " +
-                "format(\"svg\");; " +
+                "format(\"svg\"); " +
+                "unicode-range: U+0-10FFFF;; " +
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.